build.sh

```diff
@@ -428,7 +428,12 @@ fi
 perl .post-process-partial-backlink-generator.pl "$HTML_TEMP/wattsi-output/index-html" > "$HTML_OUTPUT/index.html";
 
 cp -p  entities/out/entities.json "$HTML_OUTPUT"
-cp -p "$HTML_TEMP/wattsi-output/xrefs.json" "$HTML_OUTPUT"
+if [ -f "$HTML_TEMP/wattsi-output/xrefs.json" ]; then
+  cp -p "$HTML_TEMP/wattsi-output/xrefs.json" "$HTML_OUTPUT"
+else
+  echo "$HTML_TEMP/wattsi-output/xrefs.json not found. You probably need"
```

Comment:
It seems likely that very few people will ever see this, and all in the coming month. Can we check the wattsi version and always complain if it's out of date instead?

Reply:
>  Can we check the wattsi version and always complain if it's out of date instead?

OK, raised https://github.com/whatwg/html-build/pull/146 for that. But I think it doesn’t need to be an either-or; we could merge both. This one is useful if somebody ignores the earlier warning that the other PR adds.

Reply:
Hmm, I think I tend to align with @foolip. Having extra code for this case seems a bit distracting, if we already have #146 to help guide people toward the right situation.

Reply:
OK yeah, agreed this is overkill